tox.ini

```diff
@@ -17,7 +17,7 @@ ignore = F821  ; TODO remove me when all examples in docstrings will be executab
 exclude=.venv, .git, .tox, dist, doc, build, gensim/models/deprecated
 
 [pytest]
-addopts = -rfxEXs --durations=20 --showlocals --reruns 3 --reruns-delay 1
```

Comment:
why?

Reply:
I couldn't get this to install under Py2.

Why do we need to retry on failure? Do we have any tests where this is a benefit? I've never had a test fail the first time and succeed on subsequent attempts.

Reply:
We have "blinking" tests, this trying to re-run them to fix